jerry-core/jerry.c

```diff
@@ -278,9 +256,7 @@ jerry_get_object_value (const jerry_value_t *value_p) /**< pointer to api value
 jerry_value_t
 jerry_create_void_value (void)
 {
-  jerry_value_t jerry_val;
-  jerry_val.type = JERRY_DATA_TYPE_VOID;
-  return jerry_val;
+  return ecma_make_simple_value (ECMA_SIMPLE_VALUE_EMPTY);
```

Comment:
There is a jerry_create_undefined_value. Perhaps this function could be deleted. There is "void" type anyway.